lib/muzic/src/zlib_comp.c

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright 2022 Young Mei
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "uzlib.h"
+#include "zlib.h"
+
+void outbits(struct uzlib_comp* out, unsigned long bits, int nbits);
+
+static void out4bytes(struct uzlib_comp* out,
+                      unsigned char st,
```

Comment:
What do these variable names mean? Please use descriptive variable names or document the function.